x-pack/legacy/plugins/apm/public/components/app/Main/route_config/route_names.tsx

```diff
@@ -21,5 +21,6 @@ export enum RouteName {
   SETTINGS = 'settings',
   AGENT_CONFIGURATION = 'agent_configuration',
   INDICES = 'indices',
-  SERVICE_NODES = 'nodes'
+  SERVICE_NODES = 'nodes',
+  LINK_TO_TRACE = 'linkToTrace'
```

Comment:
Our work is complete! We now officially have route names in snake_case, camelCase and kebab-case! :D
It looks like snake_case is winning, and service maps is the only odd one out. Mind changing it all to snake_case?

Reply:
😂gonna change it!!